index.js

```diff
@@ -3,35 +3,43 @@
 // console.log("%cQuestion 1", "color: red") 
 
 // console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+// => 2
+console.log("%c----------", "color: red") 
 
+console.log("%cQuestion 1", "color: red") 
+counter = 1
+counter = 2
```

Comment:
In order to declare a variable we must use let, const, or var. For the purposes of this course we are going to ignore var and just focus on let and const.  let is used when we want to have the ability to save a new value to the variable later so the answer to this problem is 
let counter = 1